docs/unocss.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig, presetUno, presetAttributify, presetIcons } from 'unocss'
+
+export default defineConfig({
+  presets: [presetUno(), presetAttributify(), presetIcons()],
+  include: [`${__dirname}/**/*`],
+  exclude: [`${__dirname}/node_modules/**/*`],
+  theme: {
+    colors: {
+      primary: {
+        DEFAULT: '#2563eb',
+        deep: '#1d4ed8',
+      },
+    },
+    breakpoints: {
+      xs: '320px',
+      sm: '640px',
+    },
```

Comment:
**Question:** does this match the breakpoint of our application?

![CleanShot 2022-03-21 at 21 50 41@2x](https://user-images.githubusercontent.com/15975785/159275170-c160eb25-5d5c-4e9c-9c3e-1f0477a96172.png)

Reply:
Updated.